src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -37,11 +37,14 @@ function isCurrentlyPausedAtBreakpoint(pause, breakpoint) {
   return bpId === pausedId;
 }
 
-function renderSourceLocation(source, line) {
+function renderSourceLocation(source, line, column) {
   const url = source.get("url") ? basename(source.get("url")) : null;
   // const line = url !== "" ? `: ${line}` : "";
   return url
-    ? dom.div({ className: "location" }, `${endTruncateStr(url, 30)}: ${line}`)
+    ? dom.div(
+        { className: "location" },
+        `${endTruncateStr(url, 30)}: ${line}${column ? `:${column}` : ""}`
```

Comment:
`column ? `:${column}` : ""` is a bit hard to follow. Can we pull this up to a local variable?